app/Services/Forge/Pipeline/OrCreateNewSite.php

```diff
@@ -55,7 +55,7 @@ private function gatherSiteData(ForgeService $service): array
             $this->information('---> Enabling site isolation.');
 
             $data['isolated'] = true;
-            $data['username'] = Str::slug($service->setting->branch);
+            $data['username'] = $service->generateUserName();
```

Comment:
I'm curios if we can use a more related method name, like `generateIsolationUserName`, or maybe something simpler.